build.gradle.kts

```diff
@@ -32,9 +34,18 @@ apply(from = rootProject.file("dependencies.gradle.kts"))
 apply(from = "scripts/git-hooks/install.gradle")
 
 measureBuilds {
-    enable = project.extra.get("measureBuildsEnabled")?.toString().toBoolean()
-    automatticProject = MeasureBuildsExtension.AutomatticProject.PocketCasts
-    authToken = project.extra.get("appsMetricsToken")?.toString()
+    enable = project.extra.properties.get("measureBuildsEnabled")?.toString().toBoolean()
+    onBuildMetricsReadyListener {
+        val report = this@onBuildMetricsReadyListener
+        SlowSlowTasksMetricsReporter.report(report)
+        runBlocking {
+            InternalA8cCiReporter.report(
+                report,
+                projectName = "pocketcasts",
+                project.extra.get("appsMetricsToken").toString(),
+            )
+        }
+    }
     attachGradleScanId = false
```

Comment:
Question (❓): I am seeing that on [JP/WPAndroid](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L123) we are having that conditionally enabled, while on [WCAndroid](https://github.com/woocommerce/woocommerce-android/blob/trunk/build.gradle#L18) we have that always enabled. Why is this always disabled for this project? 🤔

Reply:
Because this project doesn't have Gradle scans enabled - the conditions in JP/WPAndroid and WCAndroid are reflecting the existing Gradle scans configuration

Reply:
Ah okay, I looked for a Gradle enterprise configuration, and since I found one, I assumed that a Gradle scan configuration exists as well (🤷), thanks for the clarification on that Wojtek. 👍